runtime/kusama/src/lib.rs

```diff
@@ -1208,7 +1247,7 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllModules,
-	FixCouncilHistoricalVotes,
```

Comment:
This PR has mistakenly removed this migration from the next release, while it has not be removed yet.

Reply:
oh or we already have branched v27? then it is okay.

Reply:
Okay -- sorry, it is in the 27 release. Then there are two questions:

1.  is there a way to make it possible for this to also be noted in the release note? I first checked there and didn't find it, and then I checked the branch and it was there. cc @s3krit 

2. Why is the struct definition not removed? :p

Reply:
Yeah, let's add it to the v28 release notes.

"Includes a runtime migration to update session key types to be used for parachain validation and approval. All validators are encouraged to rotate their session keys following the upgrade."

Reply:
https://github.com/paritytech/polkadot/pull/2137